### DISCUSSION TITLE
Arrow function in screen

### DISCUSSION THREAD

**Discussion by gHashTag** [react-navigation/react-navigation#9901] [Q&A] [answered]
It is rumored that it is not recommended to use arrow functions when creating navigation screens, as there is a rerender problem. Can you confirm or deny this information?

**Comment by raajnadar** [comment, marked as answer]
There is no issue if you use the arrow function for navigation screens.

https://reactnavigation.org/docs/screen/#getcomponent

https://reactnavigation.org/docs/screen/#children
